Quantum-Control-Applications/Superconducting/Multi Flux Tunable transmon/Use Case 1 - Two qubit gate optimization with cryoscope/readme.md

```diff
@@ -0,0 +1,188 @@
+
+The readout resonators are pumped with an intermediate frequency generated by the OPX and upconverted with another external local oscillator using an IQ mixer.
+The reflected signal is measured by the OPX after downconversion with an IQ mixer and the local oscillator signal.
+
+The flux lines are controlled by two additional analog outputs of the OPX.
+
+In principle, the program detailed below can be easily modified if a different set-up is to be used.
+
+### 1.2 Calibration steps prior to the cryoscope and two-qubit chevron experiments
+
+Before running Cryoscope, several calibration steps are required.
+
+* The first calibration to run is the 2D readout resonator spectroscopy (flux amplitude and frequency). This spectroscopy will show the resonator frequency as a function of flux biasing, the qubit flux insensitive point (qubit frequency independent of flux bias), the qubit zero-frequency point (where the flux bias is ...), and the avoided crossing when the qubit and resonator have the same frequency. For Cryoscope, we want to set the flux line to the qubit flux insensitive point and bias it around $\phi_0/4$ during the sequence.   
+* The second important quantity to calibrate is the $\pi/2$ pulse. This can be done using a simple Rabi sequence where both the amplitude and frequency are scanned while the flux bias is set to the qubit flux insensitive point found previously.
+* (optional) If the qubit pulses are relatively short (< 100ns), then the use of optimal DRAG waveforms can enhance the fidelity of the gates [[3]](#3). 
+* The last calibration to perform is the qubit detuning versus flux bias. This will be used to validate the qubit flux insensitive point and the avoided level crossing, as well as the cryoscope measurement. The sequence can be found in the [Cryoscope use-case](#2).
+
+## 2. The configuration file
+
+### 3.1 The elements
+The configuration consists of 3 elements:
+* `qubit0` and `qubit1` send control pulses to each transmon
+* `resonator0` and `resonator1` send pump pulses and measures the reflected signal for each qubit
+* `flux_line0` and `flux_line1` send control pulses and DC voltage to the flux line of each qubit
+
+
+### 3.2 The operations, pulse and waveforms
+The qubits elements `qubit0` and `qubit1` have several operations defined that correspond to X/2, Y/2 and X gates.
+
+Since a $\pi$-pulse is 100ns long for `qubit1`, it uses a simple Gaussian envelop.
+On the other hand, since a $\pi$-pulse is 16ns long for `qubit0`, an optimized DRAG waveform [[3]](#3) is used to generate its gates.
+
+
+The `resonator` has just one readout operation defined as a constant amplitude pulse
+
+The flux line elements `flux_line0` and `flux_line1` have a single biasing operation consisting of a square pulse added on top of a DC offset set to the flux soft spot.
+
+## 3. The QUA program and data processing
+The QUA program consists of three distinct parts:
+1. Measuring the flux line step response by implementing cryoscope.

```

Comment:
Measuring the distorted

Reply:
But we don't know if it is distorted a priori, no?

Reply:
true indeed you win